.github/workflows/release.yml

```diff
@@ -9,6 +9,14 @@ on:
 
 concurrency: ${{ github.workflow }}-${{ github.ref }}
 
+env:
+  LOG_LEVEL: DEBUG
```

Comment:
Do we want it to always be set to DEBUG? Could we rely on the debug UI switch in GitHub actions?

Reply:
good catch - we don't need to default to DEBUG